UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -510,6 +510,25 @@ public void SetUpBodyPart(BodyPart bodyPart, bool instantiateCustomisations = tr
 						SetUpBodyPart(subBodyPart);
 					}
 				}
+
+				if (bodyPart.OrNull()?.OrganStorage.OrNull()?.Populater?.SlotContents != null)
+				{
+					foreach (var organ in bodyPart.OrganStorage.Populater.SlotContents)
+					{
+						if (organ == null || organ.Prefab) continue;
```

Comment:
why are we continuning if `organ.Prefab` is true?

Reply:
likely, it is missing another comparison with null.

``organ.OrNull()?.Prefab == null`` would make it better.